bin/bastion.py

```diff
@@ -48,6 +48,7 @@
 import sys
 
 import vault
+import endpoint 
```

Comment:
I'm not sure if you gain anything from running under the `bastion.py` context, especially as you don't care what the machine name is, only the domain

Reply:
True, again just thought it nice to be able to run the commands from the script. It was more of a convenience add-on than a utility add-on. But I can remove if preferred.